### PR TITLE
Use SDK types in pkg/container functions

### PR DESCRIPTION
**Pull request by alexvanin**

### CODE REVIEW

File: pkg/container/container.go [hunk 1]
```diff
@@ -69,3 +63,51 @@ func CalculateID(c *Container) *ID {
 
 	return id
 }
+
+func (c *Container) GetVersion() *pkg.Version {
```
Comment (im-kulikov) [inline pkg/container/container.go:67]: can we get rid of `Get` for `getter` methods (according to go-way)?

Comment (im-kulikov) [reply pkg/container/container.go:67]: https://golang.org/doc/effective_go.html#Getters

Comment (alexvanin) [reply pkg/container/container.go]: Yes, it is suitable there, thank you.

File: pkg/container/container.go [hunk 2]
```diff
@@ -69,3 +63,51 @@ func CalculateID(c *Container) *ID {
...
+func (c *Container) OwnerID() *owner.ID {
+	return owner.NewIDFromV2(c.Container.GetOwnerID())
+}
+
+func (c *Container) SetOwnerID(v *owner.ID) {
+	c.Container.SetOwnerID(v.ToV2())
+}
+
+func (c *Container) Nonce() []byte {
+	return c.Container.GetNonce() // return uuid?
```
Comment (cthulhu-rider) [inline pkg/container/container.go:84]: I think the question deserves a separate issue: what we do if `len(GetNonce) != uuid.Size`?

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**